rel-eng/atomic-enterprise-allinone.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Atomic Enterprise Master All-In-One
+Documentation=https://github.com/openshift/origin
+After=network.target
+After=etcd.service
+Before=atomic-enterprise-sdn-node.service
+Before=atomic-enterprise-sdn-master.service
+Requires=network.target
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/sysconfig/atomic-enterprise-allinone
+Environment=GOTRACEBACK=crash
+ExecStart=/usr/bin/atomic-enterprise start --master-config=${MASTER_CONFIG_FILE} --node-config=${NODE_CONFIG_FILE} $OPTIONS
```

Comment:
There are advantages and disadvantages for every split we do.  The biggest argument for a split is process isolation in failure domain cases - but we have rate limiters in many places already and spin looping schedulers or apiservers are also likely to kill your host.  I don't think most admins need to care about controllers via apiserver at most deployment scales (certainly up to the scales we've discussed so far), but I don't have an objection to allowing the split.